android/kotlin-central-login-oidc/app/src/main/java/com/example/app/AuthApp.kt

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2023 ForgeRock. All rights reserved.
```

Comment:
The year in the copyright note may need to be updated - i.e. to 2024... 
Let's confirm this with team the entire copyright note - for example shall we switch to "Ping Identity" instead of "ForgeRock", etc...